File: src/main/java/org/pulcifer/climate/model/wrapper/CityWrapper.java
package org.pulcifer.climate.model.wrapper;

import org.pulcifer.climate.model.City;

/**
 * Wrapper, as given by the source service.
 * It serves no purpose in this set of applications,
 * so is simply discarded upon insertion into the db.
 * As it's not used beyond the parsing of the source
 * material, it's not shared via the model project.
 */
public class CityWrapper {
    City city;

    public CityWrapper() {}

    public City getCity() {return this.city;}

    public void setCity(City city) {this.city = city; }

}

File: src/main/kotlin/org/pulcifer/climate/model/wrapper/CityWrapper.kt
package org.pulcifer.climate.model.wrapper

import org.pulcifer.climate.model.City

/**
 * Wrapper, as given by the source service.
 * It serves no purpose in this set of applications,
 * so is simply discarded upon insertion into the db.
 * As it's not used beyond the parsing of the source
 * material, it's not shared via the model project.
 */
class CityWrapper {
    var city: City? = null
}